pset-iterators.js

```diff
@@ -66,6 +72,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter((firstNameJames)=>{
+  return firstNameJames.president.split(" ")[0] === "James"
+})
+
+// console.log(presidentsNamedJames);
+
+const onlyFirstJames = presidentsNamedJames.map((firstName)=>{
+  return firstName.president
+})
```

Comment:
Very great job here! Looks like you filtered through the array of objects to get a new array of objects, then you mapped through that to get an array of names. Nice and clean!